tools/test.py

```diff
@@ -150,11 +150,15 @@ def main():
         if args.out:
             print(f'\nwriting results to {args.out}')
             mmcv.dump(outputs, args.out)
-        kwargs = {} if args.options is None else args.options
+        kwargs = cfg.get('evaluation', {})
+        kwargs.pop('interval', None)
+        kwargs.pop('gpu_collect', None)
+        kwargs.update({} if args.options is None else args.options)
+        kwargs['metric'] = args.eval
```

Comment:
looks like everything from `evaluation` is popped either overridden.

Reply:
I would introduce metrics with parameters. I don't like an idea to simply put a `score_thr` for `f1` score into `evaluation`. It is not clear that `score_thr` corresponds to `f1`. It is hard to change `score_thr` via CLI. I would propose something like that https://github.com/openvinotoolkit/mmdetection/commit/32ecee264f3a4da69e9a3dc616ff5d25cecdbf24